### ISSUE TITLE
Repeated "Discarded bad propagated block" with eth.syncing "false"

### ISSUE THREAD

**Issue by sonulrk** [ethereum/go-ethereum#14626]
#### System information

Geth version: `1.6.5`
OS & Version: Windows 7 x64
Commit hash : (if `develop`)

#### Expected behaviour
After
geth --rpc --fast --cache 8192

geth should be in sync mode

#### Actual behaviour
eth.syncing is false
and geth is giving following error as below

#### geth attach output
> admin.peers
[{
    caps: ["eth/62", "eth/63", "par/1", "par/2"],
    id: "2ecee00caef4a44ee34d4e61d45261ac225139cd13f8ed48c8d6e593c114db0ac943d94a2bda559493d815ce31d98fb7280519bca56298110f07812d83df8af2",
    name: "Parity/v1.6.7-beta-e128418-20170518/x86_64-linux-gnu/rustc1.17.0",
    network: {
      localAddress: "192.168.43.174:59870",
      remoteAddress: "123.213.75.57:30303"
    },
    protocols: {
      eth: {
        difficulty: 354764963356726900000,
        head: "0x7cce60b4b6abbf0512a1c3f7794b76585ed66a6c48adb11c5f21cb15d2c9aa9a",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "55bce301d570f3197fe625fc64088ce375f1be9ca9d228589b0f3c069f9579fde335bd04ff91bd691f194e46aa23ebe168866927378308245c7ff5ac416051e1",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:58619",
      remoteAddress: "129.78.110.127:2118"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "73664724343fc5f9f5509d40b188de70b636c079dbd0a8932f4345030615a9b833a615df6290ce19f830e495825ff79302adc6bc0eeee2384bd6d6cccbf4eb6c",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:55750",
      remoteAddress: "129.78.110.127:2119"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "7bbd3bfa20072551a86a3bd9da2f5e130936337e75e261520531a81ee17bd876b595a2a109ac5a96fa2fa44caa98923e19ba7ad6a11252c9ef9f005173fea596",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:58235",
      remoteAddress: "129.78.110.127:2113"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "9e131795e51bc76505bd2bb7e6c9b8df4c10a995e4276973d09fe4f262d2ff37bfe12ee4d24c46e0675dbf480127601853a988476261eded7677691dd1beda8b",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:55752",
      remoteAddress: "129.78.110.127:2114"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "a2e80de351e036141f969466765d4548beebc7e2b4050e1e6f7baba10a29c8631caa73ddbd952b65785eb9b2245e9f4bbec8dd84c37dd63caebf8a013685da97",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:56963",
      remoteAddress: "129.78.110.127:2120"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "b468da21fdaef9d2c31eb1ced6c720d793421f4c299bad937ac3cfab35700374c321ca5e17ca2ee3d97807b834e5de7dd1cd1a1c60e0d63d46358fb267f6792d",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:59006",
      remoteAddress: "129.78.110.127:2111"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "de6353f4b4b296a7658bb3ae7d1de26a9db6ffab5d819c420fcf1989093058eaccdf5bf3863528fb9b8ee43b4bfd3c60f4559bb43d3afdaa258b8edb41684e0d",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:58341",
      remoteAddress: "129.78.110.127:2117"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}, {
    caps: ["eth/63"],
    id: "edcd816e95e0b7373fbb501f671b0042cf19c548c5e7b0bf5a01f738708a46f45310931514a13fd87b74ad4d948a9af22b0dbe10529d12bbe9e49a45def7598d",
    name: "Geth/v1.6.6-unstable/linux-amd64/go1.7.3",
    network: {
      localAddress: "192.168.43.174:55230",
      remoteAddress: "129.78.110.127:2112"
    },
    protocols: {
      eth: {
        difficulty: 17179869184,
        head: "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        version: 63
      }
    }
}]
>

#### Backtrace

INFO [06-15|14:18:18] Imported new state entries               count=1    elapsed=1ms       processed=10094116 pending=81683
WARN [06-15|14:18:21] Discarded bad propagated block           number=1       hash=0e0e25…ebcad2
WARN [06-15|14:18:21] Discarded bad propagated block           number=1       hash=0e0e25…ebcad2
WARN [06-15|14:18:21] Discarded bad propagated block           number=1       hash=0e0e25…ebcad2
INFO [06-15|14:18:22] Imported new state entries               count=384  elapsed=284.016ms processed=10094500 pending=82474
INFO [06-15|14:18:22] Imported new state entries               count=2    elapsed=2.000ms   processed=10094502 pending=82480
INFO [06-15|14:18:23] Imported new state entries               count=1    elapsed=0s        processed=10094503 pending=82481
INFO [06-15|14:18:24] Imported new block receipts              count=888  elapsed=349.019ms number=3350463 hash=662449…e04ae4 ignored=0
INFO [06-15|14:18:25] Imported new state entries               count=384  elapsed=257.014ms processed=10094887 pending=83333
INFO [06-15|14:18:27] Imported new block receipts              count=177  elapsed=81.004ms  number=3350640 hash=123e30…ec9a7a ignored=0
INFO [06-15|14:18:30] Imported new state entries               count=384  elapsed=1.132s    processed=10095271 pending=84160
INFO [06-15|14:18:30] Imported new block receipts              count=6    elapsed=3.000ms   number=3350646 hash=2d5538…3af00d ignored=0
INFO [06-15|14:18:31] Imported new block headers               count=0    elapsed=17.001ms  number=3352783 hash=74999b…2d8f3c ignored=576
INFO [06-15|14:18:31] Imported new state entries               count=384  elapsed=229.013ms processed=10095655 pending=84973
INFO [06-15|14:18:40] Imported new state entries               count=1    elapsed=0s        processed=10095656 pending=84976
WARN [06-15|14:18:54] Synchronisation failed, retrying         err="state data download canceled (requested)"
WARN [06-15|14:22:23] Ancestor below allowance                 peer=91b4019c967f3260 number=1139648 hash=39b04e…85a283 allowance=3260646
WARN [06-15|14:22:23] Synchronisation failed, dropping peer    peer=91b4019c967f3260 err="retrieved ancestor is invalid"
WARN [06-15|14:22:45] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:23:16] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:23:39] Discarded bad propagated block           number=1       hash=eaa67c…c39e72
WARN [06-15|14:23:41] Discarded bad propagated block           number=1       hash=eaa67c…c39e72
WARN [06-15|14:23:42] Discarded bad propagated block           number=1       hash=eaa67c…c39e72
WARN [06-15|14:23:51] Discarded bad propagated block           number=1       hash=0e0e25…ebcad2
WARN [06-15|14:23:56] Discarded bad propagated block           number=1       hash=f1e26d…06b62d
WARN [06-15|14:23:59] Discarded bad propagated block           number=1       hash=f1e26d…06b62d
WARN [06-15|14:24:05] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:24:56] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:26:40] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:27:21] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:29:00] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:30:26] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:30:27] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:31:48] Synchronisation failed, retrying         err="header processing canceled (requested)"
WARN [06-15|14:32:33] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:35:13] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:35:24] Discarded bad propagated block           number=1       hash=687cfc…759e2f
WARN [06-15|14:35:24] Discarded bad propagated block           number=1       hash=687cfc…759e2f
WARN [06-15|14:35:24] Discarded bad propagated block           number=1       hash=687cfc…759e2f
WARN [06-15|14:35:25] Discarded bad propagated block           number=1       hash=687cfc…759e2f
WARN [06-15|14:35:49] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:36:53] Discarded bad propagated block           number=1       hash=c6bed4…3f2387
WARN [06-15|14:36:53] Discarded bad propagated block           number=1       hash=c6bed4…3f2387
WARN [06-15|14:36:53] Discarded bad propagated block           number=1       hash=c6bed4…3f2387
WARN [06-15|14:36:54] Discarded bad propagated block           number=1       hash=c6bed4…3f2387
WARN [06-15|14:36:57] Discarded bad propagated block           number=1       hash=2b3ea9…d1e994
WARN [06-15|14:36:57] Discarded bad propagated block           number=1       hash=2b3ea9…d1e994
WARN [06-15|14:36:57] Discarded bad propagated block           number=1       hash=2b3ea9…d1e994
WARN [06-15|14:37:02] Discarded bad propagated block           number=1       hash=05d251…23b567
WARN [06-15|14:37:02] Discarded bad propagated block           number=1       hash=05d251…23b567
WARN [06-15|14:37:02] Discarded bad propagated block           number=1       hash=05d251…23b567
WARN [06-15|14:37:02] Discarded bad propagated block           number=1       hash=05d251…23b567
INFO [06-15|14:38:00] Imported new state entries               count=1    elapsed=15.000ms  processed=10095657 pending=17
INFO [06-15|14:38:01] Imported new state entries               count=2    elapsed=20.001ms  processed=10095659 pending=49
INFO [06-15|14:38:02] Imported new state entries               count=4    elapsed=18.001ms  processed=10095663 pending=114
INFO [06-15|14:38:03] Imported new block headers               count=0    elapsed=4.000ms   number=3350838 hash=a51a38…4ea173 ignored=192
INFO [06-15|14:38:03] Imported new state entries               count=11   elapsed=45.002ms  processed=10095674 pending=267
INFO [06-15|14:38:06] Imported new state entries               count=24   elapsed=87.005ms  processed=10095698 pending=642
INFO [06-15|14:38:10] Imported new block receipts              count=3    elapsed=2.000ms   number=3350649 hash=8fb55c…d21431 ignored=0
INFO [06-15|14:38:10] Imported new state entries               count=37   elapsed=108.006ms processed=10095735 pending=1219
INFO [06-15|14:38:12] Imported new block receipts              count=5    elapsed=3.000ms   number=3350654 hash=66dbf4…4a9aaa ignored=0
INFO [06-15|14:38:12] Imported new state entries               count=51   elapsed=102.005ms processed=10095786 pending=1683
INFO [06-15|14:38:13] Imported new block receipts              count=5    elapsed=1.000ms   number=3350659 hash=827eaa…388adb ignored=0
INFO [06-15|14:38:14] Imported new state entries               count=85   elapsed=155.008ms processed=10095871 pending=2690
INFO [06-15|14:38:14] Imported new block receipts              count=3    elapsed=1.000ms   number=3350662 hash=29f217…4d43f7 ignored=0
INFO [06-15|14:38:16] Imported new block receipts              count=12   elapsed=6.000ms   number=3350674 hash=9bdad7…fc2b87 ignored=0
INFO [06-15|14:38:17] Imported new state entries               count=154  elapsed=196.011ms processed=10096025 pending=4043
INFO [06-15|14:38:20] Imported new block receipts              count=16   elapsed=7.000ms   number=3350690 hash=24d348…7dd3ed ignored=0
INFO [06-15|14:38:26] Imported new state entries               count=218  elapsed=211.012ms processed=10096243 pending=5365
INFO [06-15|14:38:32] Imported new state entries               count=239  elapsed=228.013ms processed=10096482 pending=8103
INFO [06-15|14:38:37] Imported new block receipts              count=52   elapsed=20.001ms  number=3350742 hash=2f05a1…d9522f ignored=0
INFO [06-15|14:38:38] Imported new state entries               count=282  elapsed=264.015ms processed=10096764 pending=11993
INFO [06-15|14:38:43] Imported new block receipts              count=35   elapsed=17.000ms  number=3350777 hash=8f94fd…9fe904 ignored=0
INFO [06-15|14:38:46] Imported new state entries               count=336  elapsed=343.019ms processed=10097100 pending=16033
INFO [06-15|14:38:49] Imported new block receipts              count=33   elapsed=19.001ms  number=3350810 hash=df5ea8…d48c98 ignored=0
INFO [06-15|14:38:54] Imported new state entries               count=380  elapsed=370.021ms processed=10097480 pending=21045
INFO [06-15|14:38:54] Imported new block receipts              count=28   elapsed=11.000ms  number=3350838 hash=a51a38…4ea173 ignored=0
INFO [06-15|14:38:57] Imported new state entries               count=384  elapsed=352.020ms processed=10097864 pending=26797
INFO [06-15|14:38:59] Imported new state entries               count=384  elapsed=349.019ms processed=10098248 pending=32608
INFO [06-15|14:39:05] Imported new state entries               count=384  elapsed=435.024ms processed=10098632 pending=34348
INFO [06-15|14:39:08] Imported new state entries               count=384  elapsed=472.027ms processed=10099016 pending=36577
INFO [06-15|14:39:08] Imported new block headers               count=0    elapsed=46.002ms  number=3352374 hash=214242…7cfea8 ignored=1536
INFO [06-15|14:39:12] Imported new state entries               count=384  elapsed=335.019ms processed=10099400 pending=41696
INFO [06-15|14:39:17] Imported new block receipts              count=42   elapsed=19.001ms  number=3350880 hash=c87347…d9f1df ignored=0
INFO [06-15|14:39:18] Imported new state entries               count=384  elapsed=360.020ms processed=10099784 pending=46287
INFO [06-15|14:39:23] Imported new block receipts              count=22   elapsed=10.000ms  number=3350902 hash=5678ea…67e608 ignored=0
INFO [06-15|14:39:24] Imported new state entries               count=384  elapsed=474.027ms processed=10100168 pending=47426
INFO [06-15|14:39:25] Imported new block receipts              count=11   elapsed=5.000ms   number=3350913 hash=0d1bda…062b7b ignored=0
INFO [06-15|14:39:27] Imported new block receipts              count=48   elapsed=23.001ms  number=3350961 hash=0816a4…f2f022 ignored=0
INFO [06-15|14:39:29] Imported new state entries               count=384  elapsed=517.029ms processed=10100552 pending=48672
INFO [06-15|14:39:29] Imported new block headers               count=0    elapsed=23.001ms  number=3353142 hash=7b836a…81014b ignored=768
INFO [06-15|14:39:29] Imported new block receipts              count=22   elapsed=9.000ms   number=3350983 hash=a00c90…43f0d4 ignored=0
INFO [06-15|14:39:30] Imported new block receipts              count=35   elapsed=10.000ms  number=3351018 hash=2694ea…738d16 ignored=0
INFO [06-15|14:39:32] Imported new state entries               count=384  elapsed=463.026ms processed=10100936 pending=49989
INFO [06-15|14:39:35] Imported new block receipts              count=86   elapsed=36.002ms  number=3351104 hash=89148b…45891d ignored=0
INFO [06-15|14:39:37] Imported new state entries               count=384  elapsed=403.023ms processed=10101320 pending=52933
INFO [06-15|14:39:40] Imported new block receipts              count=6    elapsed=2.000ms   number=3351110 hash=f15e89…a0cf09 ignored=0
INFO [06-15|14:39:41] Imported new state entries               count=384  elapsed=431.024ms processed=10101704 pending=55077
INFO [06-15|14:39:47] Imported new block receipts              count=295  elapsed=104.006ms number=3351405 hash=ecd0e2…9fbb40 ignored=0
INFO [06-15|14:39:49] Imported new state entries               count=384  elapsed=451.025ms processed=10102088 pending=57732
INFO [06-15|14:39:49] Imported new block headers               count=0    elapsed=17.001ms  number=3353718 hash=a6c346…3894ad ignored=576
INFO [06-15|14:39:56] Imported new block receipts              count=164  elapsed=62.003ms  number=3351569 hash=6b2b0a…b0eadd ignored=0
INFO [06-15|14:39:57] Imported new state entries               count=384  elapsed=411.023ms processed=10102472 pending=60554
INFO [06-15|14:40:07] Imported new block receipts              count=207  elapsed=77.004ms  number=3351776 hash=fe32dd…e27237 ignored=0
INFO [06-15|14:40:08] Imported new state entries               count=384  elapsed=452.025ms processed=10102856 pending=63336
INFO [06-15|14:40:19] Imported new state entries               count=384  elapsed=428.024ms processed=10103240 pending=66124
INFO [06-15|14:40:22] Imported new block receipts              count=27   elapsed=10.000ms  number=3351803 hash=9dfe1b…6fabc3 ignored=0
INFO [06-15|14:40:32] Imported new state entries               count=50   elapsed=61.003ms  processed=10103290 pending=66440
INFO [06-15|14:40:34] Imported new block receipts              count=186  elapsed=139.008ms number=3351989 hash=a86da5…a38c0c ignored=0
INFO [06-15|14:40:35] Imported new state entries               count=5    elapsed=5.000ms   processed=10103295 pending=66481
INFO [06-15|14:40:38] Imported new state entries               count=384  elapsed=396.022ms processed=10103679 pending=71640
WARN [06-15|14:40:38] Synchronisation failed, retrying         err="block body download canceled (requested)"
WARN [06-15|14:41:50] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:43:40] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:43:43] Discarded bad propagated block           number=1       hash=69ef8d…19c222
WARN [06-15|14:43:43] Discarded bad propagated block           number=1       hash=69ef8d…19c222
WARN [06-15|14:43:59] Discarded bad propagated block           number=1       hash=c6bed4…3f2387
WARN [06-15|14:45:07] Discarded bad propagated block           number=1       hash=2b3ea9…d1e994
WARN [06-15|14:45:52] Discarded bad propagated block           number=1       hash=687cfc…759e2f
WARN [06-15|14:46:47] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:46:52] Discarded bad propagated block           number=1       hash=eaa67c…c39e72
INFO [06-15|14:47:55] Imported new state entries               count=1    elapsed=2.000ms   processed=10103680 pending=17
INFO [06-15|14:48:00] Imported new block headers               count=0    elapsed=4.000ms   number=3313013 hash=909a74…ec01c5 ignored=192
INFO [06-15|14:48:01] Imported new state entries               count=1    elapsed=2.000ms   processed=10103681 pending=17
INFO [06-15|14:48:02] Imported new block headers               count=0    elapsed=2.000ms   number=3313117 hash=1d5a5b…86ca6e ignored=104
INFO [06-15|14:48:03] Imported new state entries               count=3    elapsed=12.000ms  processed=10103684 pending=65
INFO [06-15|14:48:05] Imported new state entries               count=5    elapsed=15.000ms  processed=10103689 pending=146
INFO [06-15|14:48:07] Imported new state entries               count=9    elapsed=14.000ms  processed=10103698 pending=289
INFO [06-15|14:48:08] Imported new state entries               count=17   elapsed=30.001ms  processed=10103715 pending=562
INFO [06-15|14:48:09] Imported new state entries               count=43   elapsed=56.003ms  processed=10103758 pending=1247
INFO [06-15|14:48:11] Imported new state entries               count=102  elapsed=109.006ms processed=10103860 pending=2879
INFO [06-15|14:48:15] Imported new state entries               count=192  elapsed=194.011ms processed=10104052 pending=5942
WARN [06-15|14:48:21] Synchronisation failed, retrying         err="block body download canceled (requested)"
WARN [06-15|14:50:13] Discarded bad propagated block           number=1       hash=28bc7e…659276
WARN [06-15|14:50:13] Discarded bad propagated block           number=1       hash=b3062a…388286
WARN [06-15|14:50:13] Discarded bad propagated block           number=1       hash=0c271e…4539f9
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=08efdd…1ead06
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=08efdd…1ead06
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=6b0fbd…cd571a
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=399cb7…c13c65
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=22b15e…839331
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=b3062a…388286
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=08efdd…1ead06
WARN [06-15|14:50:14] Discarded bad propagated block           number=1       hash=6b0fbd…cd571a
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=6b0fbd…cd571a
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=399cb7…c13c65
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=399cb7…c13c65
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=0c271e…4539f9
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=28bc7e…659276
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=22b15e…839331
WARN [06-15|14:50:15] Discarded bad propagated block           number=1       hash=28bc7e…659276
WARN [06-15|14:50:16] Discarded bad propagated block           number=1       hash=b3062a…388286
WARN [06-15|14:50:16] Discarded bad propagated block           number=1       hash=6b0fbd…cd571a
WARN [06-15|14:50:16] Discarded bad propagated block           number=1       hash=0c271e…4539f9
WARN [06-15|14:50:16] Discarded bad propagated block           number=1       hash=22b15e…839331
WARN [06-15|14:50:16] Discarded bad propagated block           number=1       hash=0c271e…4539f9
WARN [06-15|14:50:17] Discarded bad propagated block           number=1       hash=22b15e…839331
WARN [06-15|14:50:17] Discarded bad propagated block           number=1       hash=399cb7…c13c65
WARN [06-15|14:50:23] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:51:39] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:52:13] Discarded bad propagated block           number=1       hash=205ff5…9a950f
WARN [06-15|14:52:13] Discarded bad propagated block           number=1       hash=205ff5…9a950f
WARN [06-15|14:52:13] Discarded bad propagated block           number=1       hash=205ff5…9a950f
WARN [06-15|14:52:13] Discarded bad propagated block           number=1       hash=205ff5…9a950f
WARN [06-15|14:52:52] Discarded bad propagated block           number=1       hash=eaa67c…c39e72
WARN [06-15|14:53:05] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:53:05] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:53:05] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:53:06] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:53:06] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:53:07] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:53:07] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|14:54:00] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:55:40] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:55:54] Discarded bad propagated block           number=1       hash=205ff5…9a950f
WARN [06-15|14:56:04] Discarded bad propagated block           number=1       hash=02f1b8…733791
WARN [06-15|14:56:04] Discarded bad propagated block           number=1       hash=02f1b8…733791
WARN [06-15|14:56:04] Discarded bad propagated block           number=1       hash=02f1b8…733791
WARN [06-15|14:56:05] Discarded bad propagated block           number=1       hash=02f1b8…733791
WARN [06-15|14:56:11] Discarded bad propagated block           number=1       hash=22b15e…839331
WARN [06-15|14:56:13] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|14:56:21] Discarded bad propagated block           number=1       hash=399cb7…c13c65
WARN [06-15|14:56:28] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:56:54] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|14:57:10] Discarded bad propagated block           number=1       hash=b3062a…388286
WARN [06-15|14:57:43] Discarded bad propagated block           number=1       hash=28bc7e…659276
WARN [06-15|14:57:43] Ancestor below allowance                 peer=17a3b02809d7de63 number=1094135 hash=f69499…288e8c allowance=3261989
WARN [06-15|14:57:43] Synchronisation failed, dropping peer    peer=17a3b02809d7de63 err="retrieved ancestor is invalid"
WARN [06-15|14:58:22] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|14:58:45] Discarded bad propagated block           number=1       hash=205ff5…9a950f
WARN [06-15|15:00:17] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|15:01:16] Discarded bad propagated block           number=1       hash=b7fab8…dc826b
WARN [06-15|15:01:16] Discarded bad propagated block           number=1       hash=b7fab8…dc826b
WARN [06-15|15:01:17] Discarded bad propagated block           number=1       hash=b7fab8…dc826b
WARN [06-15|15:01:17] Discarded bad propagated block           number=1       hash=b7fab8…dc826b
WARN [06-15|15:01:22] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|15:01:23] Discarded bad propagated block           number=1       hash=6b0fbd…cd571a
WARN [06-15|15:01:24] Discarded bad propagated block           number=1       hash=08efdd…1ead06
WARN [06-15|15:01:40] Discarded bad propagated block           number=1       hash=56c8de…bce76b
WARN [06-15|15:02:30] Synchronisation failed, retrying         err="block download canceled (requested)"
WARN [06-15|15:03:28] Discarded bad propagated block           number=1       hash=22b15e…839331
WARN [06-15|15:03:28] Discarded bad propagated block           number=1       hash=806db3…7613a7
WARN [06-15|15:04:32] Discarded bad propagated block           number=1       hash=02f1b8…733791
WARN [06-15|15:05:49] Discarded bad propagated block           number=1       hash=1f11c1…817ffa
WARN [06-15|15:05:49] Discarded bad propagated block           number=1       hash=1f11c1…817ffa
WARN [06-15|15:05:49] Discarded bad propagated block           number=1       hash=1f11c1…817ffa
WARN [06-15|15:05:50] Discarded bad propagated block           number=1       hash=1f11c1…817ffa

**Comment by sonulrk** [comment]
Problem is with net speed.

**Comment by gozandris** [comment]
Same problem for me also.
Geth version: 1.6.5
OS & Version: Windows 7 x64

**Comment by sonulrk** [comment]
Try to use ssd and reconnect to fast and reliable connection.

**Comment by glynn-farrow** [comment]
Hi .. same .. but my network is very good and stable ...
Please advise

**Comment by sonulrk** [comment]
I thought it was normal but after 10 days continuous it is failed. In my case, when I got near the latest block several millions continuous "Imported new state entries" forced me to stop the process (see this issue #14647). And when I restart geth in full mode it start from number=1 :(. 

My advice NEVER EVER sync with geth in fast mode if your network cannot sync in few hours. Else who knows it is waste of time. Use geth in full mode or use parity in warp (now it is default) mode. But remember geth in full mode will take many weeks.

I exported from last try of geth full mode till block 2468704 and imported it in parity and started parity from this point. Let's see what happens.
